library_generation/generate_repo.py

```diff
@@ -48,18 +48,17 @@ def generate_from_yaml(
 
     for library_path, library in repo_config.libraries.items():
         print(f"generating library {library.get_library_name()}")
-
         generate_composed_library(
             config_path=config_path,
             config=config,
             library_path=library_path,
             library=library,
             output_folder=repo_config.output_folder,
             versions_file=repo_config.versions_file,
+            gapic_repo=not config.contains_common_protos(),
```

Comment:
Why do we call this `gapic_repo`? I see that `config` is already being passed around, can we just use `config.contains_common_protos()`? I feel it is easier to understand.

Reply:
I removed this parameter as we already pass the config.